libs/blocks/marquee/marquee.css

```diff
@@ -321,12 +326,12 @@
 }
 
 .marquee.large .text .action-area {
-  order: 2;
+  flex-direction: column-reverse;
   margin-bottom: var(--spacing-s);
 }
 
 .marquee.large .text .body-xl {
-  order: 3;
```

Comment:
Do we no longer need the `order` at all?

Reply:
No, I don't believe we need this property any more. It was previously used to position the action CTA buttons above the body text on mobile devices but the new mockups are indicating that the buttons should now fall in their original order.